image/scanner/rhel/Dockerfile

```diff
@@ -1,12 +1,12 @@
 ARG BASE_REGISTRY=registry.access.redhat.com
-ARG BASE_IMAGE=ubi8-minimal
-ARG BASE_TAG=8.6
+ARG BASE_IMAGE=ubi9-minimal
+ARG BASE_TAG=9.0.0
 
 FROM ${BASE_REGISTRY}/${BASE_IMAGE}:${BASE_TAG} AS extracted_bundle
 
 COPY bundle.tar.gz /
 WORKDIR /bundle
-RUN microdnf install tar gzip && tar -zxf /bundle.tar.gz
+RUN microdnf -y install tar gzip && tar -zxf /bundle.tar.gz
```

Comment:
Microdnf has "--assumeyes" enabled by default, from help:

Notes:
  The "--assumeyes" option is turned on by default. To switch it to an interactive prompt, specify "assumeyes=0" in the configuration file.

Reply:
Disregard, guess that changed in ubi9.

Reply:
```
% docker run -it registry.redhat.io/ubi9/ubi-minimal:latest 
[root@91d47a3f89c5 /]# microdnf install tar

(microdnf:14): librhsm-WARNING **: 21:22:38.654: Found 0 entitlement certificates

(microdnf:14): librhsm-WARNING **: 21:22:38.656: Found 0 entitlement certificates
Downloading metadata...
Downloading metadata...
Downloading metadata...
Package                                                                                                                                             Repository                                                 Size
Installing:                                                                                                                                                                                                        
 tar-2:1.34-3.el9.x86_64                                                                                                                            ubi-9-baseos                                           915.0 kB
Transaction Summary:
 Installing:        1 packages
 Reinstalling:      0 packages
 Upgrading:         0 packages
 Obsoleting:        0 packages
 Removing:          0 packages
 Downgrading:       0 packages
Is this ok [y/N]: 
```